ballerina-tests/http-misc-tests/tests/http_header_test.bal

```diff
@@ -276,3 +276,42 @@ function testPassthruWithBody() returns error? {
     }
 }
 
+type Headers record {|
+    @http:Header { name: "X-API-VERSION" }
```

Comment:
Can we use an empty header or nil value to headers, will it return a compilation error?

Reply:
We can use, it will not return a compilation error, the field name will be used as the header name. Will add a test case for that

Reply:
Added via [3b117aa](https://github.com/ballerina-platform/module-ballerina-http/pull/2153/commits/3b117aa021e089b4234b509b273b360aee9df9ff)